cypress.config.js

```diff
@@ -1,8 +1,28 @@
 const { defineConfig } = require('cypress')
+const { faker } = require('@faker-js/faker');
```

Comment:
You should use `require('faker')` instead of `require('@faker-js/faker')`. The package name is incorrect.